earthly

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 set -e
 
-bindir="$HOME/.earthly"
+bindir="$HOME/.earth"
```

Comment:
Why not use `.earthly` for this too?

Reply:
good catch, this was a mistaken search/replace.